core/arch/x86/proc.c

```diff
@@ -355,8 +357,14 @@ proc_init_arch(void)
                       (!proc_has_feature(FEATURE_FXSR) && !proc_has_feature(FEATURE_SSE)),
                   "Unsupported processor type: SSE and FXSR must match");
 
+    /* XXX i#1312: this will be set based on processor and OS feature support. */
     num_simd_saved = MCXT_NUM_SIMD_SLOTS;
     num_simd_registers = MCXT_NUM_SIMD_SLOTS;
+    /* XXX i#1312: this will not be assigned based on feature support. It represents the
```

Comment:
But isn't this already finished and needs no more work?  Why is there a XXX comment?  Please spell out what remains to be done.

Reply:
You're right. I've clarified the comments. The first one was changed to "TODO". The second one is a final comment w/o XXX/TODO.